Graphir.API/DataLoaders/DataTypeResolvers.cs

```diff
@@ -74,4 +74,13 @@ public class DataTypeResolvers
 
     public static ResourceReference? GetReferenceValue(DataType? data)
         => data is not null && data.TypeName == "Reference" ? data as ResourceReference : null;
+    
+    public static Attachment? GetAttachmentValue(DataType? data)
```

Comment:
Should use generic `GetValue<Attachment>()` instead.
This has already been implemented in PR #87